src/cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -141,6 +141,7 @@ static gpi_objtype_t to_gpi_objtype(int32_t vpitype) {
         case vpiGate:
         case vpiPrimTerm:
         case vpiGenScope:
+        case vpiPackage:
```

Comment:
Not really a module, but then again, neither are some of these other things.  Please advise if there is a better / more correct way.

Reply:
As I'm hacking away it appears that it may be easier to add a `GPI_PACKAGE` to deal with the delimiter special casing for GPI object and SimHandle creation.  However, I don't think the fallout is too bad.